ui/raidboss/data/03-hw/trial/ravana-ex.txt

```diff
@@ -1,107 +1,174 @@
-277.00 "Surpanakha 2/4" # /Ravana uses Surpanakha./
-279.00 "Surpanakha 3/4" # sync /Ravana uses Surpanakha./ window 1
-281.00 "Surpanakha 4/4" # /Ravana uses Surpanakha./
-283.00 "Orbs" # /Ravana uses The Rose of Conviction./
-# 287.39 "Rose of Hate" duration 3.61 sync /Ravana starts using The Rose Of Hate on Ravana./
-295.00 "Surpanakha 1/5" sync /Ravana uses Surpanakha./ window 1.5
-297.00 "Surpanakha 2/5" # /Ravana uses Surpanakha./
-299.00 "Surpanakha 3/5" # sync /Ravana uses Surpanakha./ window 1
-301.00 "Surpanakha 4/5" # /Ravana uses Surpanakha./
-303.00 "Surpanakha 5/5" # /Ravana uses Surpanakha./
- 
-# Scorpion Phase (Sword Charges)
- 
-308.00 "Scorpion Avatar" sync /Ravana uses Scorpion Avatar./ window 10
-313.00 "Blades of Carnage and Liberation" # sync /Ravana uses Blades of Carnage and Liberation./ window 5
-314.91 "Swift Liberation" duration 17 sync /Ravana starts using Swift Liberation on Ravana./ window 5
-356.00 "Blades of Carnage and Liberation" sync /Ravana uses Blades of Carnage and Liberation./ window 5
-358.77 "1st Double Prey" duration 17 sync /Ravana starts using Final Liberation on Ravana./ window 5
-384.00 "2nd Double Prey"
-390.00 "Outer AOE" sync /Ravana uses Final Liberation./ window 3
-396.00 "Inner AOE" # /Ravana uses Final Liberation./
- 
-# Dragonfly Phase
- 
-405.00 "Dragonfly Avatar" sync /Ravana uses Dragonfly Avatar./ window 200
-# 406.35 "Warlord Shell" duration 2.65 sync /Ravana starts using Warlord Shell on Ravana./
-409.00 "Warlord Shell" sync /Ravana uses Warlord Shell./ window 5
-411.00 "The Seeing" duration 3 sync /Ravana readies The Seeing/ window 10
-417.00 "Atma-Linga" # /Ravana uses Atma-Linga./
-418.00 "The Seeing" duration 3 sync /Ravana readies The Seeing/
-430.00 "Blinding Blade" sync /Ravana uses Blinding Blade./ window 5
-433.00 "Atma-Linga" # /Ravana uses Atma-Linga./
-435.00 "Tapasya" sync /Ravana uses Tapasya./ window 5
-442.00 "Atma-Linga" # /Ravana uses Atma-Linga./
-445.00 "Blinding Blade" # /Ravana uses Blinding Blade./
-448.00 "Atma-Linga" # /Ravana uses Atma-Linga./
-450.00 "Atma-Linga" # /Ravana uses Atma-Linga./
-452.00 "Tapasya" sync /Ravana uses Tapasya./ window 5
-458.00 "Blinding Blade" # /Ravana uses Blinding Blade./
-461.00 "Atma-Linga" # /Ravana uses Atma-Linga./
-470.00 "Blinding Blade" # /Ravana uses Blinding Blade./
-473.00 "Atma-Linga" # /Ravana uses Atma-Linga./
-475.00 "Tapasya" sync /Ravana uses Tapasya./ window 5
\ No newline at end of file
+# -ii EC1 EC2 EA5 EAA EBE 1299 13BC 13BF 140F
+# ic "Spirit Gana" "Moon Gana" "Chandrahas"
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+# During all phases, any attacks will increase Ravana's Bloodlust meter.
+# The relationship between increases and damage done is not fully understood.
+# The meter also increases passively during all phases except the opening block.
+# Any displayed phase lengths are the maximum that could occur with no "extra" Bloodlust.
```

Comment:
Overall it doesn't *really* matter, it just means phases will be shorter than the timeline thinks. I *think* the meter is incremented by a lot if Ravana loses more than X% HP within Y time, but I haven't dug too deeply, since it doesn't make much of a difference.